<?xml version="1.0"?>
<launch>
    <!-- Load parameters -->
    <node name="find_grasppoint_server" pkg="grasping_pipeline" type="find_grasppoint_action_server.py"
        args= "$(find grasping_pipeline)/models/" output="screen"/>
</launch>